### PR TITLE
Tomas' solutions for #22

### PR DESCRIPTION
**Pull request by TomasEliz**

### CODE REVIEW

File: tomas/index.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
+let log = console.log;
+
+// Check if two given integer values are in the range 50 to 99 (this includes both 50 and 99). Print true if either of them are in the range
+let x = 1;
+let y = 100;
+let result;
+if((x < 50 || x > 99) && (y < 50 || y > 99)) {
```
Comment (ciamac-da) [inline tomas/index.js]: Your integer variable are in range of  < 49.999 or bigger than 99! It should be in range of 50 to 99!!
You also have to check if EITHER of them are in the range not both of them!

File: tomas/index.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+let x2 = 5;
+let y2 = 3;
+if((x2 + y2) == 8 || (x2 - y2) == 8) {
+    log('6 -->', true);
+}
+
+// Check whether either one of two integers is 15, or if their sum is 15. If one of these is the case, print true
+let x3 = 7;
+let y3 = 8;
+if((x3 != 15 && y3 != 15) || (x3 + y3 == 15)) {
```
Comment (ciamac-da) [inline tomas/index.js]: You have to check if x3 or y3 are 15

File: tomas/index.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+// Calculate the sum of the two given integers. If the two values are the same, then print triple their sum
+let x5 = 3;
+let y5 = 3;
+let sum = x5 + y5;
+if(x5 === y5) {
+    log('triple sum =', sum * 3);
+}
+
+// Calculate the difference between a specified number and 19. Print double the difference if the specified number is greater than 19
+let z2 = 22;
```
Comment (ciamac-da) [inline tomas/index.js:80]: The result is correct but make sure that you use Math.abs() method to calculate the difference.

**Review by ciamac-da — CHANGES_REQUESTED** [review]

**Comment by TomasEliz** [comment]
I see. Well I guess I understood it the wrong way. I thought EITHER OF =
NONE OF.  And that's what I checked. I got confused thinking that maybe if
you'd want me to check if x or y is part of some group, the exercise would
state "CHECK IF ONE OF THOSE" is that and that.

**Review by ciamac-da — APPROVED** [review]

**Pull request merged**